src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorTest.groovy

```diff
@@ -100,7 +102,7 @@ class IqPolicyEvaluatorTest
 
     then: 'performs a remote scan'
       1 * RemoteScannerFactory.getRemoteScanner("appId", "stage", ["*.jar"], [], workspace,
-          proprietaryConfig, _ as Logger, 'instance-id') >> remoteScanner
+          proprietaryConfig, _ as Logger, 'instance-id', advancedProperties) >> remoteScanner
```

Comment:
I don't understand why this test takes advancedProperties? What makes it different from the others?

Reply:
Same for the ones below.